include/vapor/DC.h

```diff
@@ -2107,6 +2116,10 @@ class VDF_API DC : public Wasp::MyBase {
     //
     virtual bool getDimension(string dimname, DC::Dimension &dimension) const = 0;
 
+    //! \copydoc GetDimension()
+    //
+    virtual bool getDimension(string dimname, DC::Dimension &dimension, long ts) const { return getDimension(dimname, dimension); };
```

Comment:
ts in unused. How come?

Reply:
DCs do not support time-varying dimensions by default.